_episodes/14-supplemental-rstudio.md

```diff
@@ -40,16 +40,21 @@ our computer, we choose the option "Existing Directory":
 > where your Git executable is, and you won't be able to progress further
 > in this lesson until you tell RStudio where it is.  
 > ### Find your Git Executiable
-> First make sure that you have installed either [Git](https://git-scm.com/downloads/)
-> or [GitHub](https://desktop.github.com/) on your computer.
-> Next, open your shell on Mac or Linux, or on Windows open the command prompt.
+> First let's make sure that Git is installed on your computer.
+> Open your shell on Mac or Linux, or on Windows open the command prompt
 > and then type:
 > - `which git` (Mac, Linux)
 > - `where git` (Windows)
-> Copy the path to the git executable (e.g. On one Windows computer which had
-> GitHub Desktop installed on it, the path was:
+>
+> If there is no version of Git on your computer, please either install [Git](https://git-scm.com/downloads/)
```

Comment:
How about referring to https://carpentries.github.io/workshop-template/#git here so that we do not need to maintain extra instructions here at all? https://swcarpentry.github.io/git-novice/setup.html is already doing that as well.

Reply:
Which [can not be recovered](https://en.wikipedia.org/wiki/Sunk_cost) unfortunately, and therefore should not be part of the decision when what gets merged ;-)

I don't mean this harshly at all; Please don't get me wrong! I'm grateful to @richmccue for alerting us to the  problem and for providing a fix :-) Just pointing out that previous maintainers choose to refer to the setup info that's maintained centrally.

Will test this on the weekend and report back.

Reply:
Thank you for the helpful wikipedia link. 

I agree that this link should point to the same instructions as the setup to be consistent with the rest of the lesson.  I do not think that that merging this PR should be blocked by this link being inconsistent.